lib/Dialect/SV/Transforms/RTLCleanup.cpp

```diff
@@ -155,7 +157,9 @@ void RTLCleanupPass::runOnGraphRegion(Region &region, bool shallow) {
       // Reprocess the merged body because this may have uncovered other
       // simplifications.
       runOnGraphRegion(alwaysOp->getRegion(0), /*shallow=*/true);
-      runOnGraphRegion(alwaysOp->getRegion(1), /*shallow=*/true);
+      // AlwaysFFOp has another region, the reset block.
```

Comment:
could you loop over all the regions regardless of op type?

Reply:
Done